index.js

```diff
@@ -364,22 +316,22 @@ let Fixtures = {
             }
 
             let getRequiredFieldPromise = MetadataHandler.getRequiredFields(model.module)
-            .then((requiredFields) => {
-                _.each(requiredFields, (field) => {
-                    if (_.isUndefined(request.data[field.name])) {
-                        request.data[field.name] = MetadataHandler.generateFieldValue(field);
+                .then((requiredFields) => {
+                    _.each(requiredFields, (field) => {
+                        if (_.isUndefined(request.data[field.name])) {
+                            request.data[field.name] = MetadataHandler.generateFieldValue(field);
+                        }
+                    });
+
+                    // Populate the `credentials` object.
+                    if (model.module === 'Users') {
+                        request.data.user_name = _generateInternalUsername(request.data.user_name);
+                        _insertCredentials(request.data.user_name, request.data.user_hash);
                     }
-                });
-
-                // Populate the `credentials` object.
-                if (model.module === 'Users') {
-                    request.data.user_name = _generateInternalUsername(request.data.user_name);
-                    _insertCredentials(request.data.user_name, request.data.user_hash);
-                }
 
-                // Use chakram.post to bulk create the record(s).
-                bulkRecordCreateDef.requests.push(request);
-            });
+                    // Use chakram.post to bulk create the record(s).
+                    bulkRecordCreateDef.requests.push(request);
+                });
```

Comment:
The block of code from line 319 ~ 334 is the same as the original, except for indentation. This make this commit change more code than it needed (functionally).